code/ui/manager/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const breakpointMin600 = '(min-width: 600px)';
```

Comment:
For constants I'd prefer `SCREAMING_SNAKE_CASE` as that usually communicates constants.
```suggestion
export const BREAKPOINT_MIN_600 = '(min-width: 600px)';
```